tests/io/test_metar.py

```diff
@@ -163,6 +163,14 @@
      Metar('TFFF', 14.6, -61.0, 5, datetime(2017, 5, 11, 18, 30), 110, 19, 1000, 'RA', 'BCFG',
            np.nan, 'FEW', 1400, 'BKN', 2100, 'BKN', 2700, np.nan, np.nan, 6, 27, 24,
            units.Quantity(1015, 'hPa').m_as('inHg'), 63, 41, 0, '')),
+    # Interchanged wind and vis groups
+    ('KBMI 121456Z COR 10SM 10005KT SCT055 OVC065 23/22 A3000 RMK AO2'
+     'LTG DSNT E AND SE TSB1358E13 SLP150 6//// T02280222 53011=',
+     Metar('KBMI', 40.47, -88.92, 267, datetime(2021, 8, 12, 14, 56), np.nan, np.nan,
```

Comment:
`datetime` needs to be changed. Since reports only include day, hour, minute, we hard-code May 2017 for the rest in the tests:
```suggestion
     Metar('KBMI', 40.47, -88.92, 267, datetime(2017, 5, 12, 14, 56), np.nan, np.nan,
```

Reply:
Ahh, that's why it worked in my stand-alone version of the test but not in the decorator block. Everything is now passing on my machine!